packages/modules/promotion/integration-tests/__tests__/services/promotion-module/promotion.spec.ts

```diff
@@ -638,7 +638,7 @@ moduleIntegrationTestRunner({
               application_method: expect.objectContaining({
                 target_type: "order",
                 allocation: "across",
-                max_quantity: 0,
+                max_quantity: null,
```

Comment:
The `max_quantity` is `null` after the `Number` serializer of MikroORM was converting it to `0`